src/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "../Introduce/Introduce.scss";
+import {Link, useNavigate} from "react-router-dom";
+
+
+
+const Introduce = ()=> {
+    
+    const navigate = useNavigate();
+    const goToProduct = () => {
+        navigate("/Product");
+    }
+
+    return(
+        <nav className="Introduce">
```

Comment:
```suggestion
        <nav className="introduce">
```
컴포넌트의 가장 최상위 부모태그를 컴포넌트의 이름과 동일하게 잘 해주셨습니다
그러나 대문자로 시작하는 부분은 컴포넌트의 이름에만 적용이 됩니다.
이 경우에는 className이기 때문에 소문자로 시작해주세요!

Reply:
감사합니다. 수정하였습니다!